### PR TITLE
[gl3w] Use khrplatform.h from the egl-registry port

### PR DESCRIPTION
**Pull request by cfillion**
The gl3w port checks the hash of the downloaded file khrplatform.h since #18858. That file was modified yesterday (KhronosGroup/EGL-Registry@57b4876de0f33677ece92dd9de0ef105ce69139d), requiring the hash to be updated.

This fixes the following error when installing gl3w:

```
Starting package 41/41: gl3w:x64-windows-static
Building package gl3w[core]:x64-windows-static...
-- Downloading https://github.com/skaslev/gl3w/archive/99ed321100d37032cb6bfa7dd8dea85f10c86132.tar.gz -> skaslev-gl3w-99ed321100d37032cb6bfa7dd8dea85f10c86132.tar.gz...
-- Extracting source C:/Tools/vcpkg/downloads/skaslev-gl3w-99ed321100d37032cb6bfa7dd8dea85f10c86132.tar.gz
-- Applying patch 0001-enable-shared-build.patch
-- Using source at C:/Tools/vcpkg/buildtrees/gl3w/src/5f10c86132-7e24c6e824.clean
-- Downloading https://www.khronos.org/registry/EGL/api/KHR/khrplatform.h -> khrplatform.h...
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' = off
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Downloading https://www.khronos.org/registry/EGL/api/KHR/khrplatform.h
Error: Failed to download from mirror set:
File does not have the expected hash:
             url : [ https://www.khronos.org/registry/EGL/api/KHR/khrplatform.h ]
       File path : [ C:\Tools\vcpkg\downloads\khrplatform.h.1560.part ]
   Expected hash : [ 93d9075718eddb69c44482acdc72bbbd3511741272a6124d05ab1ef0702ef03e918501403b6fd334faf2c61f3332f34b7730158aa090db3d448c32b5dd9d9e67 ]
     Actual hash : [ 0adf658c45095bf31bb1cca4b9eb6fe944550d1ee7f95b5601494869c57b4b43f544e8160933a1b7d6d63c975de08191f27f9f16d50bf3b7c7e209f28a4bcfb5 ]
```

### CODE REVIEW

File: ports/gl3w/portfile.cmake [hunk 1]
```diff
@@ -12,7 +12,7 @@ vcpkg_from_github(
 vcpkg_download_distfile(KHRPLATFORM_H
   URLS "https://www.khronos.org/registry/EGL/api/KHR/khrplatform.h"
   FILENAME khrplatform.h
-  SHA512 93d9075718eddb69c44482acdc72bbbd3511741272a6124d05ab1ef0702ef03e918501403b6fd334faf2c61f3332f34b7730158aa090db3d448c32b5dd9d9e67
+  SHA512 0adf658c45095bf31bb1cca4b9eb6fe944550d1ee7f95b5601494869c57b4b43f544e8160933a1b7d6d63c975de08191f27f9f16d50bf3b7c7e209f28a4bcfb5
   )
```
Comment (Neumann-A) [inline ports/gl3w/portfile.cmake:16]: isn't that file provided by the port `egl-registry` ?

Comment (cfillion) [reply ports/gl3w/portfile.cmake:16]: Ah yes, it is. Then I suppose the port should install that as a dependency instead?

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for your PR!

**Comment by BillyONeal** [comment]
Thanks for the fix!

**Pull request merged**